### ISSUE TITLE
AZERTY Switch

### ISSUE THREAD

**Issue by Fr-Dae** [unitystation/unitystation#6550]
### Description
Add a switch button to set qwerty/azerty on 1 clic

![2021-05-10-191003_1920x1080_scrot](https://user-images.githubusercontent.com/23202421/118563351-b079f400-b76e-11eb-95e7-f09ea44371d7.png)

**Comment by corp-0** [comment]
I wonder what's the demand for this feature.

**Comment by Fr-Dae** [comment]
**AZERTY** -  ĄŽERTY
France(and DomTom), Belgium, Switzerland, Luxembourg, Lithuanian, Dutch, and some germans

(**Dom-Tom** : Guadeloupe,Martinique,Guyane,La Réunion,Saint-Pierre-et-Miquelon,Mayotte,Saint-Barthélemy,Saint-Martin,Wallis-et-Futuna,Polynésie française,Nouvelle-Calédonie,Terres australes et antarctiques françaises,Clipperton)

if you want to export to steam, you have to think about the accessibility and user-friendly options. now, not later


quebec, are based on a local mix of qwerty and azerty , but have azerty layout by default on all drivers



it's the same as offering options for non-binary, in caracter creator. i
t's all the interest of independent games or opensources, to be able to make choices that big triple A license would not do because it is ONLY aimed at the US market. which is not the case here. 


**I don't think that a quick switch between two keyboard preconfigs is a real technical challenge and requires a lot of time to set up.**

**Comment by NoooneyDude** [comment]
I agree, but initial values should be pulled from the player's keyboard layout.

Don't need to add UI for something that can be automated.

**Comment by Fr-Dae** [comment]
so rather than adding a switch that loads a qwerty or azerty profile, by simply pressing a button/switch, 

would you prefer to add an option, which inspects the computer and finds the default keyboard profile/layer? 

I'm not an expert, but isn't accessing the machine level much more complicated and buggy than the temporary solution I propose?

**Comment by NoooneyDude** [comment]
No.
